src/lib.rs

```diff
@@ -351,6 +351,15 @@ fn render_message(mut w: impl Write, config: &Config<'_>, msg: &Message, dbc: &D
         writeln!(&mut w, "}}")?;
         writeln!(w)?;
 
+        writeln!(&mut w, "/// Access message id")?;
+        writeln!(&mut w, "pub fn id(&self) -> u32 {{",)?;
```

Comment:
```suggestion
        writeln!(&mut w, "pub fn message_id(&self) -> u32 {{",)?;
```

For consistency with the constant name?